app/models/user.rb

```diff
@@ -5,6 +5,12 @@ class User < ApplicationRecord
   attr_accessor :remember_token, :activation_token, :reset_token
 
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: Relationship.name,
+    foreign_key: "follower_id", dependent: :destroy
```

Comment:
```suggestion
    foreign_key: :follower_id, dependent: :destroy
```
sao ở đây phải định nghĩa class_name, foreign_key ?

Reply:
Mặc định Rails sẽ suy ra tên class dựa theo association và tạo ra liên kết khoá ngoại theo tên_class_id, ở đây ta phải chỉ định tên class vì tên association không giống với class name của model, chỉ định foreign_key vì ta muốn liên kết khoá ngoại với follower_id chứ không phải "relationship_id"